src/controllers/TripController.js

```diff
@@ -69,6 +74,26 @@ class TripController {
       return Customize.errorMessage(req, res, err.message, 500);
     }
   }
-}
 
+  /**
+   * Manager should be able to view requests for approvals
+   * @static
+   * @param {object} req request object
+   * @param {object} res response object
+   * @memberof TripController
+   * @returns {object} data
+   * @description PATCH api/v1/trips/requests
+   */
+  static async getRequestsByManager(req, res) {
+    const { id } = req.user;
+    const result = await availtripRequestsToManager(id);
+    try {
+      if (result) {
+        return Customize.successMessage(req, res, 'Requests fetched successfully', result, 200);
+      }
```

Comment:
Try not to place `if condition` in the controller

Reply:
@william000000 Please explain this better?